EligibilityCheckActivity/handler.ts

```diff
@@ -64,15 +66,21 @@ export const getEligibilityCheckActivityHandler = (
         err => new Error(`Error: [${readableReport(err)}]`)
       )
     )
-      .chain(fiscalCode => {
-        return soapClientAsync
+      .chain(fiscalCode =>
+        // If the Fiscal Code is a testing one and is defined the test SOAP client,
+        // this is used instead the production SOAP client
+        fromNullable(process.env.TEST_FISCAL_CODES)
+          .map(_ => _.split(","))
+          .filter(_ => _.includes(fiscalCode))
+          .mapNullable(() => testSoapClientAsync)
+          .getOrElse(soapClientAsync)
```

Comment:
I think it'd better to move this logic in the `index`, where we build objects, and leave `handler` untouched. Or maybe even in the client itself.

Reply:
We cannot move this logic into the index because we need the `fiscalCode` that is available inside the handler. IMHO the client is better as is because is configurable and logic agnostic.

Reply:
You can wrap the method:

```ts
const client = {
  ConsultazioneSogliaIndicatore: ({ CodiceFiscale, ...rest }) => 
      testCFs.includes(CodiceFiscale) 
          ? testSoapClientAsync.ConsultazioneSogliaIndicatore({ CodiceFiscale, ...rest })
          : soapClientAsync.ConsultazioneSogliaIndicatore({ CodiceFiscale, ...rest })
}
```

Reply:
Nice suggestion :)

Reply:
Fixed in 868d4a6